### DISCUSSION TITLE
No adapters found

### DISCUSSION THREAD

**Discussion by dbeckwith** [gfx-rs/wgpu#2758] [Q&A] [answered]
`cargo run --bin wgpu-info` shows no output on my desktop machine. How can I debug this? All of the detection seems to be at runtime so I'm not sure if there's some library I'm missing or what.

NixOS 21.11
Linux 5.10.113
Graphics NVidia GTX 1070
Driver 510.60.02

**Comment by cwfitzgerald** [comment, marked as answer]
You can add `RUST_LOG=trace` which will print the most verbose level of logging. It should tell you what is going on.

**Reply by dbeckwith** [reply]
Thank you! This allowed me to find exactly which libraries it was looking for so I could add them to my Nix shell.

**Reply by cwfitzgerald** [reply]
That's great!

**Reply by adsick** [reply]
Hi, for some reason I don't get any output from "hello" example with RUST_LOG=trace, how can I fix that?
![image](https://user-images.githubusercontent.com/24528839/206725732-c58d4b64-23cb-474c-94e9-0a75004af631.png)

**Reply by adsick** [reply]
never mind, I need to do `export RUST_LOG=trace` or `RUST_LOG=trace cargo run ...`
